<?xml version="1.0"?>
<launch>

  <!-- Convert the pointcloud2 to laserscan -->
  <node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pcl_to_scan" >
    <param name="min_height" value="0.3"/>
    <param name="max_height" value="0.5"/>
    <param name="range_min" value="0.4"/>
    <param name="range_max" value="8.0"/>
    <param name="angle_max" value="3.13"/>
    <param name="angle_min" value="-3.12"/>
    <!-- <param name="use_inf" value="true"/> -->
    <param name="use_inf" value="false"/>
    <param name="target_frame" value="velodyne"/> <!-- not changing it, keeps same frame, to keep ns-->
    <param name="angle_increment" value=".01"/> <!-- 0.003 -->

    <remap from="cloud_in" to="mid/points"/>
    <remap from="scan" to="scan"/>
  </node>

</launch>